app/models/activity.rb

```diff
@@ -26,6 +26,11 @@ def project=(new_project); end
   scope :latest, -> do
     includes(:trackable).order('activities.created_at DESC').limit(20)
   end
+  scope :filter_by_user_id, -> (id) { where user_id: id }
```

Comment:
```suggestion
  scope :by_user, -> (id) { where user_id: id }
```
Same on `L30-31` to keep consistent with convention/our existing scope names